app/build.gradle

```diff
@@ -143,12 +157,13 @@ android {
         release {
             minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
-            signingConfig getUseDebugSigningOnRelease() ? debug.signingConfig : release.signingConfig
+            signingConfig signingConfigs.release
             buildConfigField 'String', 'PROPS_ENDPOINT', '"https://igalia.github.io/wolvic/props.json"'
             buildConfigField "String", "MK_API_KEY", "\"${getMKApiKey()}\""
         }
         debug {
             applicationIdSuffix getDevApplicationIdSuffix()
+            signingConfig signingConfigs.debug
```

Comment:
I don't understand this. By default we already use release for release and debug for debug. The useDebugSigningOnRelease is precisely to override that behaviour.

Reply:
Removed. That was just for local testing.